### PR TITLE
AST-2278 - Social sharing icon/options for single posts

### PR DESCRIPTION
**Pull request by nathanrodrigues2111**
### Description
Jira : https://brainstormforce.atlassian.net/browse/AST-2278

Added social sharing support

### Screenshots
https://d.pr/wMMN8z

### Types of changes
New feature (non-breaking change which adds functionality) 

### How has this been tested?
- Checked in customizer preview 

### Checklist:
- [x] My code is tested
- [ ] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request 

[AST-2278]: https://brainstormforce.atlassian.net/browse/AST-2278?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: inc/customizer/extend-custom-controls/src/social-icons/item-component.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ let svg_icons = Object.keys( astIcons )
 const ItemComponent = props => {
 
 	const Icons = window.svgIcons;
-
+
```
Comment (imnavanath) [inline inc/customizer/extend-custom-controls/src/social-icons/item-component.js:15]:
```suggestion
```

**Review by imnavanath — APPROVED** [review]

**Review by er-roshan — COMMENTED** [review]
@nathanrodrigues2111 Please resolve branch conflicts

**Review by er-roshan — APPROVED** [review]

**Review by yssambare12 — APPROVED** [review]

**Review by imnavanath — APPROVED** [review]

File: inc/customizer/configurations/typography/class-astra-single-typo-configs.php [hunk 2]
```diff
@@ -69,6 +69,9 @@ public function register_configuration( $configurations, $wp_customize ) {
 							'six'   => array(
 								'title' => __( 'Extended spacing options', 'astra' ),
 							),
+							'seven' => array(
```
Comment (imnavanath) [inline inc/customizer/configurations/typography/class-astra-single-typo-configs.php:72]:
```suggestion
							'eight' => array(
```

Comment (imnavanath) [reply inc/customizer/configurations/typography/class-astra-single-typo-configs.php:72]: As seven already present.

**Pull request merged**